man/systemd.network.xml

```diff
@@ -1188,6 +1188,16 @@
           </listitem>
         </varlistentry>
 
+        <varlistentry>
+          <term><varname>UserClass=</varname></term>
+          <listitem>
+            <para>A DHCPv4 client can use UserClass option to identify the type or category of user or applications
+            it represents. The information contained in this option is a string object that represents the user class
+            of which the client is a member. Each class sets an identifying string of information to be used by the DHCP
+            service to classify identifying clients. Takes a whitespace-separated list of user class information.</para>
```

Comment:
"classify identifying clients" → "classify clients"

"Takes a whitespace-separated list of user class information." → "Takes a whitespace-separated list."

So, RFC is "request for comments", which is particularly confusing in this context. Please fix the patch description.

So... the RFC 3004 specifies that each identifier is an opaque string. I don't see why we shouldn't do the same, i.e. do c-unescaping on the words. If we're implementing the option, let's just follow the RFC as closely as possible.

And as usual, we face the issue of resetting the option. Please make it so that multiple assignments are merged, and an empty assignment resets the list. Mention this in the documentation.

Reply:
>So... the RFC 3004 specifies that each identifier is an opaque string. I don't see why we shouldn't do the same, i.e. do c-unescaping on the words. If we're implementing the option, let's just follow the RFC as closely as possible.

The underlying API does not support they will break.

> And as usual, we face the issue of resetting the option. Please make it so that multiple assignments are merged, and an empty assignment resets the list. Mention this in the documentation.

I thought multiple assignments we pick the last one they do not merge . Same as vendor class identifier.  Yes it resets actually with a empty string.